PullRequest.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from config import GITHUB_USER, GITHUB_PW
+
+class PullRequest(object):
+    def __init__(self, event=None):
+        self.event = event
+        if event is not None:
+            self.issue_url = event['pull_request']['issue_url']
+
+    @property
+    def label_url(self):
+        return "{}/labels".format(self.issue_url)
+
+    def request_labels_json(self):
+        return requests.get(self.label_url, auth=(GITHUB_USER, GITHUB_PW)).json()
+
+    def validate_labels(self, required_any, required_all, banned):
+        self_labels_list = [l['name'] for l in self.request_labels_json()]
```

Comment:
why is this called `self_labels_list`? This could be a bit confusing since `self` refers to an instance of this class. You might want to just call it `labels list`.